src/dev-dashboard/ui/src/styles.css

```
@@ -0,0 +1,22 @@
+@import "tailwindcss";
+@import "tw-animate-css";
+@import "@ui/theme/styles.css";
+@source "../../../utils/ui";
+@import "./slate-grid.css";
```

Stylelint (17.11.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)